package.json

```diff
@@ -23,8 +23,7 @@
     "build:cowfi": "nx build cow-fi",
     "build:cosmos": "nx run cowswap-frontend:build:cosmos",
     "ci:build": "yarn run build:cowswap && yarn run build-widget",
-    "ipfs:build": "PUBLIC_URL=\".\" yarn build",
-    "ipfs:publish": "npx ipfs-deploy build -p pinata -O",
+    "ipfs": "nx run cowswap-frontend:ipfs",
```

Comment:
From what I can tell, the `ipfs` command is just doing a regular `build`.
What about removing it completely in favor of the `build` command instead?

Reply:
no, it depends on build, but it runs `npx ipfs-deploy build/cowswap -p pinata -O`